salt/_modules/kubernetes.py

```diff
@@ -1975,3 +1975,126 @@ def replace_daemonset(
             raise CommandExecutionError(exc)
     finally:
         _cleanup(**cfg)
+
+
+def node_annotations(name, **kwargs):
```

Comment:
For consistency with other annotations commands, `name` parameter should be `node_name`.

Reply:
I copy-pasted the signature of [`node_labels`](https://github.com/scality/metalk8s/blob/020703de54117108286831e813d3076ecdf39d3a/salt/_modules/kubernetes.py#L321), I'll keep it as is until we refactor this custom module (which is essentially a long list of copy-pasted functions)